app/javascript/dashboard/components-next/avatar/Avatar.vue

```diff
@@ -112,6 +118,17 @@ const containerStyles = computed(() => ({
   height: `${props.size}px`,
 }));
 
+const borderRadiusClass = computed(() => {
+  if (props.roundedFull) {
+    return 'rounded-full';
+  }
+
+  // Calculate radius as 25% of size, with minimum of 4px
+  const calculatedRadius = props.size * BORDER_RADIUS.RATIO;
+  const radius = Math.max(Math.round(calculatedRadius), BORDER_RADIUS.MIN);
+  return `rounded-[${radius}px]`;
```

Comment:
This won't work, tailwind styles cannot be built like this. You can either approximate to a few standard sizes, or else, add this as a style tag. I would prefer the approximating solution